daemon/mgr/container_utils.go

```diff
@@ -103,39 +104,46 @@ func (mgr *ContainerManager) generateName(id string) string {
 }
 
 func parseSecurityOpts(meta *ContainerMeta, securityOpts []string) error {
+	var (
+		labelOpts []string
+		err       error
+	)
 	for _, securityOpt := range securityOpts {
 		if securityOpt == "no-new-privileges" {
 			meta.NoNewPrivileges = true
 			continue
 		}
-		if err := parseSecurityOpt(meta, securityOpt); err != nil {
-			return err
+		fields := strings.SplitN(securityOpt, "=", 2)
+		if len(fields) != 2 {
+			return fmt.Errorf("invalid --security-opt %s: must be in format of key=value", securityOpt)
+		}
+		key, value := fields[0], fields[1]
+		switch key {
+		// TODO: handle other security options.
+		case "apparmor":
+			meta.AppArmorProfile = value
+		case "seccomp":
+			meta.SeccompProfile = value
+		case "no-new-privileges":
```

Comment:
prevent processes in container to gain new privileges via the `--security-opt=no-new-privileges` flag

just attach a record which may do us a favour.